samwise.js

```diff
@@ -1,4 +1,4 @@
-const doubledNumbers = numbers.map(num => num * 2);
```

Comment:
SCRIPT-The way you've refactored this complex method into simpler ones is great. It makes the code much more maintainable.